src/System.Runtime.CompilerServices.Unsafe/ref/System.Runtime.CompilerServices.Unsafe.cs

```diff
@@ -16,6 +16,7 @@ public static partial class Unsafe
         public static bool AreSame<T>(ref T left, ref T right) { throw null; }
         public unsafe static void* AsPointer<T>(ref T value) { throw null; }
         public unsafe static ref T AsRef<T>(void* source) { throw null; }
+        public static ref T AsRef<T>(ref readonly T source) { throw null; }
```

Comment:
Note: when you next update your compiler toolset, this will have to be changed to use `in` instead of `ref readonly`. I don't see the LDM notes on this posted yet, but here's [a summary](https://github.com/dotnet/roslyn/issues/22381).